kotlin/src/main/com/looker/sdk/Constants.kt

```diff
@@ -28,7 +28,7 @@ package com.looker.sdk
 
 const val ENVIRONMENT_PREFIX = "LOOKERSDK"
 const val SDK_TAG = "KT-SDK"
-const val LOOKER_VERSION = "23.18"
+const val LOOKER_VERSION = "23.17"
```

Comment:
consistent with the bad version. If this isn't going to be published I'm not sure it matters.